cfn-resources/integration-tests/util/awsHelper.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	cfn "github.com/aws/aws-sdk-go-v2/service/cloudformation"
+)
+
+func GetAWSClient() (client *cfn.Client, err error) {
+	cfg, err := config.LoadDefaultConfig(context.Background())
+	if err != nil {
+		return nil, errors.New("Error loading AWS configuration: " + err.Error())
```

Comment:
```suggestion
		return nil, fmt.Errorf("Error loading AWS configuration: %w", err)
```
https://go.dev/blog/go1.13-errors

Reply:
updated